worker/invoiceForward.js

```diff
@@ -0,0 +1,357 @@
+import {
```

Comment:
I'm leaving this file as a reference for now. I'll delete before pulling out of draft.